test/middleware.basic.js

```diff
@@ -5,8 +5,8 @@ var request = require('supertest');
 describe('middleware', function(){
   describe('.next()', function(){
     it('should behave like connect', function(done){
-      var app = express()
-        , calls = [];
+      var app = express();
```

Comment:
Express now follows `standard` formatting for all new work, so that would be no-semicolons on these lines.  To be clear, that does not mean apply it to all other lines, just the ones you are touching already.

Reply:
I would just leave this off the pr since it's a style only change.

Reply:
Good to know.